src/api_service.py

```diff
@@ -0,0 +1,15 @@
+from typing import List, Generator
+
+import requests
+
+
+def get_beststories() -> List[int]:
+    response = requests.get('https://hacker-news.firebaseio.com/v0/beststories.json')
```

Comment:
Do you think it makes sense to create a const for the host part of the URL?

Reply:
Yes, that sounds reasonable. Maybe it's even worth putting the address in env vairiable. Also I think, some parameters with docker-compose.yaml can be added to .env